website/static/js/filepage/index.js

```diff
@@ -363,6 +364,37 @@ var FileViewPage = {
         }, 1000);
 
         self.mfrIframeParent = $('#mfrIframeParent');
+        function goToRevisions(e){
+            var editable = self.editor && self.editor.selected;
+            var viewable = self.mfrIframeParent.is(':visible');
+            if (viewable){
+                self.mfrIframeParent.toggle();
+            }
+            if (editable) {
+                self.editor.selected = false;
+            }
+            self.revisions.selected = true;
+        }
+
+        function changeVersionHeader(){
+            m.render(document.getElementById('version-link'), m('a', {onclick: goToRevisions}, document.getElementById('version-link').innerHTML));
+        }
+
+        //anchor checking hack that will select if true
+        var idx = window.location.href.indexOf('?');
+        var idx_uri = (idx > 0) ? URI.parseQuery(window.location.href.slice(idx)) : '';
```

Comment:
We have a helper for parsing query params, `osfHelpers.urlParams`. Use this instead.